horovod/torch/__init__.py

```diff
@@ -127,12 +160,15 @@ def DistributedOptimizer(optimizer, named_parameters=None, compression=Compressi
         compression: Compression algorithm used during allreduce to reduce the amount
                      of data sent during the each parameter update step.  Defaults to
                      not using compression.
+        backward_passes_per_step: Number of expected backward passes to perform before
+                                  executing allreduce
```

Comment:
... before calling step()/synchronize().  This allows accumulating gradients over multiple mini-batches before executing allreduce and updating parameters.